Dijktra.py

```diff
@@ -0,0 +1,38 @@
+# Dijktra's shortest path algorithm. Prints the path from source to target.
+ 
+ 
```

Comment:
please delete the extra spaces in your code.

Reply:
Hi @aneels3 , I do the changes

Reply:
great!
It will be great if you add some algorithm problem by creating an issue.

Reply:
Nice @RobbiNespu 
It will be great if you add some algorithm problem by creating an issue.